_includes/references.md

```diff
@@ -31,6 +31,7 @@
 {% comment %}<!-- TODO: switch miniscript link to some sort of overview page when available -->{% endcomment %}
 [miniscript]: http://bitcoin.sipa.be/miniscript/miniscript.html
 [musig]: https://eprint.iacr.org/2018/068
+[output script descriptor]: https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md
```

Comment:
Is this alias reference required? Why not use the [output script descriptors] link below?

Reply:
Definitely not required, but we've had to use the other link several times already for the singular case so it seemed appropriate to just alias it:

```text
$ git grep 'output script descriptor]'
_posts/en/newsletters/2018-12-04-newsletter.md:  [output script descriptor][output script descriptors] for each address
_posts/en/newsletters/2019-02-12-newsletter.md:  specified using an [output script descriptor][output script
_posts/en/newsletters/2019-06-19-newsletter.md:  specific private keys using an [output script descriptor][output
```

I'm happy to add another case of `[output script descriptor][output script descriptors]` if you think that's cleaner.

Reply:
Got it. Let's keep the alias. Is there a way to define the alias in reference to the original [output script descriptors] reference, rather than duplicate the URL?